modules/sdk/project-templates/project-templates/build.gradle

```diff
@@ -158,6 +158,8 @@ test {
 	systemProperty "http.proxyPort", System.properties["http.proxyPort"]
 	systemProperty "maven.debug", System.properties["maven.debug"]
 	systemProperty "maven.repo.local", System.properties["maven.repo.local"]
+	systemProperty "nodejs.npm.ci.registry", project.property('nodejs.npm.ci.registry')
+	systemProperty "nodejs.npm.ci.sass.binary.site", project.property('nodejs.npm.ci.sass.binary.site')
```

Comment:
Is it possible to configure Gradle and Maven to use this custom Sass binary site? Have a look at https://github.com/sass/node-sass#binary-configuration-parameters.